libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/TestActions/SetProperties.cs

```diff
@@ -52,9 +55,27 @@ public async override Task ExecuteAsync(TestAdapter adapter, BotCallbackHandler
             {
                 await inspector((dc) =>
                 {
+                    var mockSettingsMiddleware = dc.Context.TurnState.Get<MockSettingsMiddleware>();
```

Comment:
>mockSettingsMiddleware [](start = 24, length = 22)

I think the original design where there is a mocksettings property on the test script is better.  I was confused by calling it mockProperties.  SetProperties is definitely useful for setting dialog/conversation, but the settings really needs to be setup before the dialog and it doesn't make sense to set those using SetProperties.

Reply:
Sure. I will keep them both